.github/workflows/prod_tests.yaml

```diff
@@ -0,0 +1,163 @@
+name: prod_tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "2 * * * *" # “At minute 2.” https://crontab.guru
+  # pull_request: # temporary
+  #   branches:
+  #     - trunk
+
+env:
+  SSHNP_ATSIGN: "@8incanteater"
+  SSHNP_ATSIGN_KEYS: ${{ secrets.ATKEYS_8INCANTEATER }}
+  SSHNPD_ATSIGN: "@8052simple"
+  SSHNPD_ATSIGN_KEYS: ${{ secrets.ATKEYS_8052SIMPLE }}
+  SSHRVD_ATSIGN: "@8485wealthy51"
+  SSHRVD_ATSIGN_KEYS: ${{ secrets.ATKEYS_8485WEALTHY51 }}
```

Comment:
Don't need SSH*_ATSIGN_KEYS environment variables, that's what the setup keys steps are for down below